lib/messagebird/client.rb

```diff
@@ -232,6 +232,16 @@ def message(id)
       Message.new(request(:get, "messages/#{id}"))
     end
 
+    # Retrieve messages with optional paging and status filter.
+    def message_list(limit: 10, offset: 0, status: '')
+      params = [['limit', limit], ['offset', offset]]
```

Comment:
the hash variant would be readable thou

Reply:
i missed one thing, please use options hash instead of keyword argument, as ruby 1.9.3 has no support for it. Cheers

Reply:
fixed! thank you